app/src/main/java/org/zephyrsoft/trackworktime/util/DateTimeUtil.java

```diff
@@ -37,14 +38,27 @@
 	private static final DateTimeFormatter LOCALIZED_DATE = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM);
 	private static final DateTimeFormatter LOCALIZED_TIME = DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT);
 	private static final DateTimeFormatter DATE = DateTimeFormatter.ofPattern("yyyy-MM-dd");
-	private static final DateTimeFormatter HOUR_MINUTES = DateTimeFormatter.ofPattern("HH:mm");
-	private static final DateTimeFormatter TIME_PRECISE = DateTimeFormatter.ofPattern("HH:mm:ss.SSS");
 	private static final DateTimeFormatter LOCALIZED_DAY_AND_DATE = new DateTimeFormatterBuilder()
 			.appendPattern("eeee")
 			.appendLiteral(", ")
 			.appendLocalized(FormatStyle.SHORT, null)
 			.toFormatter();
 
+	/** E.g. Fri, 12.9 */
+	private static final DateTimeFormatter LOCALIZED_DAY_AND_SHORT_DATE =
+			createLocalizedDayAndShortDateFormat();
+
+	private static DateTimeFormatter createLocalizedDayAndShortDateFormat() {
+		String shortDate = DateTimeFormatterBuilder.getLocalizedDateTimePattern(
+				FormatStyle.SHORT,
+				null,
+				IsoChronology.INSTANCE,
+				Locale.getDefault()
+		).replaceAll("[ /.-]? *[yY]+ *[ 年/.-]?", ""); // Remove year and year-separators
```

Comment:
This is a bit of magic, but it's pretty much impossible to get only localized day and month